app/views/hyrax/transfers/new.html.erb

```diff
@@ -1,23 +1,11 @@
 <h1>Transfer ownership of "<%= @proxy_deposit_request.to_s %>"</h1>
 <span class="sr-only">Select a user to transfer <%= @proxy_deposit_request.to_s %> to, add optional comments and then press transfer.</span>
-<%= form_for @proxy_deposit_request,
-             url: hyrax.work_transfers_path(@work),
-             html: { class: 'form-horizontal', id: 'new_transfer' } do |f| %>
-  <%= error_messages_for(f.object) %>
-  <div class="control-group">
-    <%= f.label :transfer_to, "User", class: 'control-label' %>
-    <div class="controls">
-      <%= f.text_field :transfer_to, "aria-hidden" => true %>
-    </div>
-  </div>
-  <div class="control-group">
-    <%= f.label :sender_comment, "Comments", class: 'control-label' %>
-    <div class="controls">
-    <%= f.text_area :sender_comment %>
-    </div>
-  </div>
+<%= simple_form_for @proxy_deposit_request,
+             url: hyrax.work_transfers_path(@work) do |f| %>
+  <%= f.input :transfer_to %>
+  <%= f.input :sender_comment %>
 
   <div class="form-actions">
-    <%= f.submit "Transfer", class: 'btn btn-primary', data: {confirm: "Are you sure you want to transfer ownership of this work to another user? Click Ok to transfer or Cancel to return to the transfer screen"} %>
+    <%= f.submit class: 'btn btn-primary', data: { confirm: "Are you sure you want to transfer ownership of this work to another user? Click Ok to transfer or Cancel to return to the transfer screen" } %>
```

Comment:
@mjgiarlo I'm concerned that google translate isn't going to do a good job with this.  Happy to give it a go though.